search_service/proxy/elasticsearch.py

```diff
@@ -157,8 +157,11 @@ def _get_search_result(self, page_index: int,
                 results.append(model(**result))
             except Exception:
                 LOGGING.exception('The record doesnt contain specified field.')
-
-        return search_result_model(total_results=response.hits.total,
+        if isinstance(response.hits.total, dict):
```

Comment:
might be useful to add a comment why this might happend. dict is what ES 7 returns, ES 6 returns int